Libraries/Lists/FlatList.js

```diff
@@ -688,7 +688,6 @@ class FlatList<ItemT> extends React.PureComponent<Props<ItemT>, void> {
         removeClippedSubviews={removeClippedSubviewsOrDefault(
           _removeClippedSubviews,
         )}
-        accessibilityRole="table" // TODO(macOS GH#774): Inherit NSAccessibilityTable protocols
```

Comment:
Could we do something like this?

```
...(Platform.OS === 'macos' && {accessibilityRole="table"})
```

Reply:
I had second thoughts about forcing Flatlist to behave like NSTable, should we leave that decision to clients?

Reply:
Fair